WowPacketParser/Hotfix/IHotfixSerializer.cs

```diff
@@ -61,13 +63,14 @@ static IHotfixSerializer()
 
             using (var stringLocal = serializationEmitter.DeclareLocal<string>())
             {
-                serializationEmitter.LoadConstant(isString ? @"""{0}"", " : "{0}, ");
+                serializationEmitter.LoadConstant(isString ? @"'{0}', " : "{0}, ");
                 serializationEmitter.LoadArgument(0); // instance
                 serializationEmitter.CallVirtual(propInfo.GetGetMethod());
                 if (isString)
                 {
-                    serializationEmitter.LoadConstant(@"\""");
-                    serializationEmitter.LoadConstant(@"""");
+                    serializationEmitter.Call(stringEscape);
+                    serializationEmitter.LoadConstant(Environment.NewLine);
```

Comment:
Wait what. Why would you have newlines put out (also newlines in double quoted strings are fine in SQL)

Reply:
It's to unify all SQL output.
We used to use single quote: '
And make newlines \n
Also we escaping strings by using MySqlHelper